sources/packages/api/src/constants/error-code.constants.ts

```diff
@@ -75,3 +75,23 @@ export const STUDENT_ACCOUNT_CREATION_MULTIPLES_SIN_FOUND =
  */
 export const STUDENT_ACCOUNT_CREATION_FOUND_SIN_WITH_MISMATCH_DATA =
   "STUDENT_ACCOUNT_CREATION_FOUND_SIN_WITH_MISMATCH_DATA";
+/**
+ * An error was found during the offering validation and the process
```

Comment:
There are 2 sentences in this comment. I am not able to get the connection between both the sentences.

Reply:
I will update as below. Please let me know if it is more clear.
![image](https://user-images.githubusercontent.com/61259237/189423426-128b7150-7560-483e-977c-f56e96193b3e.png)

Reply:
Changed.